docs/en/ingest-management/fleet/upgrade-elastic-agent.asciidoc

```diff
@@ -130,22 +133,49 @@ subscriptions].
 [[view-upgrade-status]]
 == View upgrade status
 
-. On the **Agents** tab, when you trigger an upgrade, agents that are upgrading
- have the status `Updating` until the upgrade is complete, then the status
- changes back to `Healthy`.
-+
+On the **Agents** tab, when you trigger an upgrade, agents that are currently 
+upgrading show a detailed upgrade status indicator beside the agent version.
+
 [role="screenshot"]
-image::images/upgrade-status.png[Screenshot of Agents tab showing upgrade status]
+image::images/upgrade-states.png[Detailed state of an upgrading agent]
+
+The following table explains the various possible upgrade states.
+
+.{agent} upgrade states
+|===
+| State | Description
+
+| Upgrade downloading | {agent} is downloading the archive containing the new version artifact.
```

Comment:
How about listing the upgrade states in the order they happen? These can be found [here](https://github.com/elastic/elastic-agent/issues/3119).